### DISCUSSION TITLE
agent configuration

### DISCUSSION THREAD

**Discussion by V1dal9** [utmstack/UTMStack#885] [Q&A] [answered]
How do I configure the utmstack agent on my syslog server so that it only fetches the logs I want? What is the agent's configuration file for the inputs?

**Comment by c3s4rfred** [comment, marked as answer]
Hi @V1dal9, enter to your instance, then go to Integrations menu, once there check the integration you want, inside each one are the steps described to configure. If your integrations is Syslog -> 
![image](https://github.com/user-attachments/assets/c43b17cf-99c3-46cb-9092-e1fe7006a1b8)
that's a generic syslog integration, so, the only way to fetch the logs you want is stop sending them by that integration. If you want to receive a specific kind of logs, check if your device in one of the supported integrations.

Best regards
